docs/dependencies.md

```diff
@@ -1,80 +1,196 @@
 - `brew unlink gcc-arm-none-eabi-49` (example)
 - `brew link --overwrite gcc-arm-none-eabi-53` (example)
 - `arm-none-eabi-gcc --version` (should now say v5.3.1)
 
-#### 2. Make
+**Windows:**
+
+Install from the download page.
+
+### GNU Make
+
+| Confirmed version | Download page |
+|:-----------------:|:-------------:|
+| 4.1 | [sourceforge.net](http://gnuwin32.sourceforge.net/packages/make.htm) |
+
 In order to turn your source code into binaries, you will need a tool called `make`. Windows users need to explicitly install `make` on their machines. Make sure you can use it from the terminal window.
 
-Download and install the latest version from: http://gnuwin32.sourceforge.net/packages/make.htm
+| OS | Distribution | Package Manager | Command |
+|:-- |:------------ |:--------------- |:------- |
+| Linux | Debian | apt-get | `sudo apt-get install make` |
+| OSX | n/a | [Homebrew](http://brew.sh/) | `brew install make` |
+| OSX | n/a | [Macports](http://www.macports.org) | `port install make` |
+| Windows | n/a | n/a | `# See detailed instructions below` |
+
+**Windows:**
+
+Install from the download page.
+
+### 32-bit Cyclic Redundency Check Utility
+
+`crc32` - Calculates CRC sums of 32 bit length
+
+| Confirmed version | Download page |
+|:-----------------:|:-------------:|
+| ? | N/A |
+
+| OS | Distribution | Package Manager | Command |
+|:-- |:------------ |:--------------- |:------- |
+| Linux | Debian | apt-get | `sudo apt-get install libarchive-zip-perl` |
+| OSX | n/a | n/a | `# Available by default` |
+| Windows | n/a | n/a | `# See detailed instructions below` |
+
+**Windows:**
+
+Install `MinGW` and add it to your path.
+
+### XXD Hex Dump Utility
+
+`xxd` - tool to make (or reverse) a hex dump
 
-#### 3. Device Firmware Upgrade Utilities
-Install dfu-util 0.8. Mac users can install dfu-util with [Homebrew](http://brew.sh/) `brew install dfu-util` or [Macports](http://www.macports.org), Linux users may find it in their package manager, and everyone can get it from http://dfu-util.gnumonks.org/index.html
+| Confirmed version | Download page |
+|:-----------------:|:-------------:|
+| 1.10 | N/A |
 
-#### 4. Zadig
-In order for the device to show up on the dfu list, you need to replace the USB driver with a utility called [Zadig](http://zadig.akeo.ie/). Here is a [tutorial](https://community.spark.io/t/tutorial-installing-dfu-driver-on-windows/3518) on using it. This is only required for Windows users.
+| OS | Distribution | Package Manager | Command |
+|:-- |:------------ |:--------------- |:------- |
+| Linux | Debian | apt-get | `sudo apt-get install xxd` |
+| OSX | n/a | n/a | `# Available by default` |
+| Windows | n/a | n/a | `# See detailed instructions below` |
 
-#### 5. Git
+**Windows:**
 
-Download and install Git: http://git-scm.com/
+Install `MinGW` and add it to your path.
 
-#### 6. Command line tools
+### Zadig (Windows only)
```

Comment:
In theory this should no longer be required as we implement proper WCID descriptors for Windows on both Gen 2 and Gen 3 devices (but not for Core). So we might add something like 'generally this should not be required for recent releases'.